pkg/chaosdaemon/util/util.go

```diff
@@ -46,11 +46,11 @@ func ReadCommName(pid int) (string, error) {
 }
 
 // GetChildProcesses will return all child processes's pid. Include all generations.
-// only return error when /proc/pid/tasks cannot be read
+// only return error when /proc/pid/stat cannot be read
```

Comment:
It's not accurate here either, it should be `/proc`.

```suggestion
// Only return error when `/proc` cannot be read.
```